### PR TITLE
Capstone 1 project - HTML CSS and JavaScript

### PR DESCRIPTION
**Pull request by Recillah-Khamala**
This pull request contains my first capstone project, a website showcasing my mastery of HTML, CSS, and JavaScript.
Below is a loom video link showing my description of the project.
https://www.loom.com/share/bc9ad1c8feb94243a9cc2d69ba13e764

### CODE REVIEW

File: html/about.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HTML Hyperlinks</title>
+    <link rel="stylesheet" href="../style.css">
+</head>
+
+<body>
+    <section id="about">
```
Comment (CollinsTatang) [inline html/about.html]: - [ ] Please add the header section to the about page
### **Solution**
```
<header>
        <div class="menu">
            <img src="../images/Xicon.svg" class="xicon" alt="normal button" />
            <ul>
                <li class="menu-li">
                    <a href="../html/home.html" class="nav-link">
                        <img src="../images/logo1.svg" width="350" height="65" alt="icon" />
                    </a>
                </li>
                <li class="menu-li">
                    <a href="../html/about.html" class="nav-link">About</a>
                </li>
                <li class="menu-li">
                    <a href="#program" class="nav-link">Program</a>
                </li>
                <li class="menu-li">
                    <a href="#join" class="nav-link">Join</a>
                </li>
                <li class="menu-li">
                    <a href="#sponsor" class="nav-link">Sponsor</a>
                </li>
                <li class="menu-li">
                    <a href="#news" class="nav-link">News</a>
                </li>
            </ul>
        </div>
        <nav class="header mobile">
            <div class="nav-top">
                <ul>
                    <li>
                        <i class="fa-brands fa-facebook-f"></i>
                    </li>
                    <li>
                        <i class="fa-brands fa-twitter"></i>
                    </li>
                    <li>English</li>
                    <li>Language</li>
                    <li>Logout</li>
                </ul>
            </div>
            <div class="nav-bottom">
                <a href="../html/home.html">
                    <img src="../images/logo.svg" width="350" height="65" alt="icon" />
                </a>
                <ul class="nav-bottom-ul">
                    <li><a href="../html/about.html">About</a></li>
                    <li><a href="#program">Program</a></li>
                    <li><a href="#join">Join</a></li>
                    <li><a href="#sponsor">Sponsor</a></li>
                    <li><a href="#news">News</a></li>
                    <li><a href="#" class="cc-campaign">LT Campaign</a></li>
                </ul>
            </div>
        </nav>
    </header>

```

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+                    <h4>Network</h4>
+                    <p class="program-about">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Reiciendis,
+                        libero cumque voluptatibus repellat
+                    </p>
+                </li>
+            </ul>
+            <a class="join none"> Join LT Global Summit 2022 </a>
+            <p class="see-projects none1"><a>SEE THE WHOLE PROJECTS</a></p>
+        </section>
```
Comment (CollinsTatang) [inline html/index.html]: Please try to match the main program section as on the [figmal design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Figmal Design**
figMain
### **Your Design**
YourMain

File: style.css [hunk 3]
```diff
@@ -0,0 +1,635 @@
...
+  /* width: 86%; */
+
+  /* margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white; */
+}
+
```
Comment (CollinsTatang) [inline style.css]: - [ ] Please remove all commented code from your codebase to make your code look clean and easy to maintain.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,635 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.speakers .spk-hr {
+  width: 48%;
+  margin: 6px auto 54px auto;
+}
+
```
Comment (CollinsTatang) [inline style.css:377]: Please style your speaker section to look almost like the of the [Figmal Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
### **Figmal Design**
figSpeaker

### **Your Design**
YoSpeaker

File: html/about.html [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                <img src="/images/daumkakao.jpg" width="100" height="50" alt="daumkakao" />
+                <img src="/images/airbnb.png" width="100" height="50" alt="airbnb" />
+            </div>
+        </section>
+    </section>
+    <script src="../js/mMenu.js"></script>
+    <script src="../js/programSpeakers.js"></script>
+</body>
+
+</html>
```
Comment (CollinsTatang) [inline html/about.html]: - [ ]Kindly redesign the about page to match that of the [figmal design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
###**Figmal Design**
about

File: README.md [hunk 6]
```diff
@@ -1,2 +1,44 @@
...
+
+## Built With
+
+- HTML 
+- CSS
+- JavaScript
+
+## Live Demo (if available)
+
+[Live Demo Link](http://rn486.github.io/Capstone1-project/)
```
Comment (CollinsTatang) [inline README.md:16]: - [ ] Please your live demo link has a broken CSS style link so it display only the HTML structure page
### [live demo](https://rn486.github.io/Capstone1-project/)
live

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Rn486, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 6 above)
Comment (Recillah-Khamala) [reply README.md:16]: Thank you for the comments, but I would like pointers on the deployment part. I have not found a solution to make it visible.

Comment (CollinsTatang) [reply README.md:16]: Ok, 
Please can we have a call to check it out?

Comment (Recillah-Khamala) [reply README.md:16]: Yes, please. Thank you.

Comment (CollinsTatang) [reply README.md:16]: Join Zoom Meeting
https://microverse.zoom.us/j/93366219619?pwd=eXRiNWIwRWl3QjRSTHdBTTVwVXZvUT09

Meeting ID: 933 6621 9619
Passcode: 925314
One tap mobile
+13017158592,,93366219619#,,,,*925314# US (Washington DC)
+13126266799,,93366219619#,,,,*925314# US (Chicago)

Dial by your location
        +1 301 715 8592 US (Washington DC)
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 386 347 5053 US
        +1 564 217 2000 US
        +1 646 931 3860 US
        +1 669 444 9171 US
        +1 669 900 6833 US (San Jose)
        +1 929 205 6099 US (New York)
        +1 253 215 8782 US (Tacoma)
Meeting ID: 933 6621 9619
Passcode: 925314
Find your local number: https://microverse.zoom.us/u/adgv23LQHT

**Review by omarsalem7 — APPROVED** [review]
Hi @Rn486 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- Live demo work correctly ✔️
- Descriptive pull request ✔️
- Project has met all the technical aspects you needed for this project. Keep rocking!  ✔️


Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] It will be better if you increase the `font-size` in the desktop version for `<p>`
- [ ] [OPTIONAL] It will be better if you instructions on how to clone the project and open it in the README file.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**